dockers/docker-orchagent/ipinip.json.j2

```diff
@@ -83,7 +83,8 @@
     {
         "TUNNEL_DECAP_TABLE:IPINIP_TUNNEL" : {
             "tunnel_type":"IPINIP",
-            "dscp_mode":"uniform",
+            "dscp_mode":"pipe",
```

Comment:
Is `pipe` mode supported on all platforms?
And for warm-reboot scenario, the `ipinip.json` is not reloaded, meaning the new configuration will not be applied. Any plan for this scenario?